src/Pos/Wallet/Light/Redirect.hs

```diff
@@ -0,0 +1,121 @@
+{-# LANGUAGE TypeFamilies #-}
+
+-- | Various monadic redirects implementing wallet constraints. Mostly stubs.
+
+module Pos.Wallet.Light.Redirect
```

Comment:
Wtf is that?
1. Why isn't it in `lwallet`?
2. We don't use redirects nowadays.